src/EventListener/ConfigListener.php

```diff
@@ -161,6 +161,8 @@ private function setGlobals()
             $usersCur = null;
         }
 
+        $twig->addGlobal('bolt_name', Version::name());
+        $twig->addGlobal('bolt_version', Version::VERSION);
```

Comment:
Ah, I see.. They snuck in from here.. https://github.com/bolt/bolt/blob/release/3.2/src/EventListener/ConfigListener.php#L165-L167

I'll remove them.